sst/include/SimengMemInterface.hh

```diff
@@ -0,0 +1,125 @@
+
+#include <sst/core/sst_config.h>
+#include <sst/core/interfaces/stdMem.h>
+#include <sst/core/eli/elementinfo.h>
+
+#include <chrono>
+#include <cmath>
+#include <cstring>
+#include <iomanip>
+#include <iostream>
+#include <string>
+#include <map>
+#include <vector>
+#include <set>
+#include <type_traits>
+
+#include  "simeng/MemoryInterface.hh"
+#include "simeng/span.hh"
+
+using namespace simeng;
+using namespace SST::Interfaces;
+
+namespace SST {
+    namespace SSTSimeng {
+        class SimengMemInterface: public MemoryInterface {
```

Comment:
Comment explaining the class similar to existing header files

Reply:
PTAL